### PR TITLE
python3Packages.crccheck: 0.6 -> 1.0

### PR DESCRIPTION
**Pull request by etu**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/crccheck/default.nix [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0ckymm6s5kw08i1j35fy2cfha1hyq94pq1kc66brb552qgjs91jn";
-    extension = "zip";
+    sha256 = "1ay9lgy80j7lklm07iw2wq7giwnv9fbv50mncblqlc39y322vi0p";
   };
 
+  disabled = !isPy3k;
+
   meta = with stdenv.lib; {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/crccheck/default.nix:17]:
```suggestion
  meta = with lib; {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please fix the merge conflict.

**Comment by etu** [comment]
Rebased on master :) 

I had forgot about this PR.

File: pkgs/development/python-modules/crccheck/default.nix [hunk 2]
```diff
@@ -1,18 +1,20 @@
 { lib, stdenv, buildPythonPackage, fetchPypi
+{ lib, stdenv, buildPythonPackage, fetchPypi, isPy3k
 , nose }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/crccheck/default.nix:3]:
```suggestion
{ lib, stdenv, buildPythonPackage, fetchPypi, isPy3k, nose }:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by etu** [comment]
@ofborg eval

**Comment by etu** [comment]
That eval error doesn't look like my fault, rebasing and reevaling.

@ofborg eval

**Pull request closed without merging**